share/instruments/instruments.xml

```diff
@@ -10946,6 +10946,29 @@
                   <longName>Double Bass</longName>
                   <shortName>Db.</shortName>
             </Instrument>
+            <Instrument id="octobass">
+                  <longName>Octobass</longName>
+                  <shortName>Ob.</shortName>
+                  <description>Octobass</description>
+                  <musicXMLid>strings.octobass</musicXMLid>
+                  <transposingClef>F</transposingClef>
+                  <concertClef>F15mb</concertClef>
+                  <barlineSpan>1</barlineSpan>
+                  <aPitchRange>21-31</aPitchRange>
+                  <pPitchRange>12-33</pPitchRange>
+                  <transposeDiatonic>-14</transposeDiatonic>
+                  <transposeChromatic>-24</transposeChromatic>
+                  <Channel name="arco">
+                        <program value="43"/> <!--Contrabass-->
+                  </Channel>
+                  <Channel name="pizzicato">
+                        <program value="32"/> <!--Accoustic Bass-->
+                  </Channel>
+                  <Channel name="tremolo">
+                        <program value="44"/> <!--Tremolo Strings-->
+                  </Channel>
+                  <genre>orchestra</genre>
```

Comment:
No need to do this, octobass is very rarely used even in orchestra.

Reply:
True, it may not need this indeed. But if any genre, then Orchestra

Reply:
Not every instrument needs a genre/genres :-)

Reply:
If even octobass is in, then why not alto flute or clarinet in C? I mean, there're a lot of them which are more common

Reply:
OK